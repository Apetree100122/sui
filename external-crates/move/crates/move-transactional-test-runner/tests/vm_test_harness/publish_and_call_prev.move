//# publish
module 0x40::m {
    public fun next(n: u64): u64 {
        n + 1
    }
}

//# publish-and-call --call 0x42::m::a --call 0x40::m::next 1
module 0x42::m {
    use 0x40::m::next;

    fun a(): u64 {
        next(10)
    }
}
