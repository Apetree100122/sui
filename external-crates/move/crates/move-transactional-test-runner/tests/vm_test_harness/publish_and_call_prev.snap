---
source: crates/move-transactional-test-runner/src/framework.rs
---
processed 2 tasks

task 1, lines 8-15:
//# publish-and-call --call 0x42::m::a --call 0x40::m::next 1
return values: 11
return values: 2
